pkg/dcmtime/common.go

```diff
@@ -0,0 +1,147 @@
+package dcmtime
+
+import (
+	"errors"
+	"fmt"
+	"strconv"
+	"strings"
+	"time"
+)
+
+// isIncluded returns whether `check` is included in `limit`.
+//
+// Example: to test whether seconds should be included, you would:
+// isIncluded(PrecisionSeconds, [caller-passed-limit])
+func isIncluded(check PrecisionLevel, precision PrecisionLevel) bool {
+	return check <= precision
+}
+
+// truncateMilliseconds truncate nanosecond time.Time value to arbitrary precision.
+func truncateMilliseconds(
+	nanoSeconds int, precision PrecisionLevel,
```

Comment:
nit: not sure if this needs to be a broken line, as per our other discussions on the other PR--let's not break lines unless they are exceedingly long to the point that it hurts readability as per the loose Go convention.


https://github.com/golang/go/wiki/CodeReviewComments#line-length

> In other words, break lines because of the semantics of what you're writing (as a general rule) and not because of the length of the line. If you find that this produces lines that are too long, then change the names or the semantics and you'll probably get a good result.